### DISCUSSION TITLE
Re-order the audio devices in the pop-up menu/settings.

### DISCUSSION THREAD

**Discussion by DuVious** [File-New-Project/EarTrumpet#1586] [Ideas]
Currently, the audio devices are fixed - meaning you cannot change the order. This is a problem if you have multiple audio devices on your computer, as you'll have to scroll either halfway down, all the way down, to a specific spot, etc.

![image](https://github.com/user-attachments/assets/af15be53-b1d4-498d-8254-6dcfcaecee46)
As seen here, I have a lot of audio devices. I would need to scroll downwards 4-5 times to reach the audio device I _actually_ use.
If you were able to either drag an audio device to the top of the list or change the order by using the built-in settings, it would make life much more bearable - you wouldn't need to scroll down EVERY time you open eartrumpet!

**Comment by axg20202** [comment]
+1. Even better would be a setting that allows customizing devices that are shown/not shown in the list as well as their order in the list.
